modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -532,9 +567,29 @@ RenderablePointCloud::ColorSettings::ColorSettings(const ghoul::Dictionary& dict
     , enableOutline(EnableOutlineInfo, false)
     , outlineColor(OutlineColorInfo, glm::vec3(0.23f), glm::vec3(0.f), glm::vec3(1.f))
     , outlineWeight(OutlineWeightInfo, 0.2f, 0.f, 1.f)
+    , outlineStyle(OutlineStyleInfo)
+    , applyCmapToOutline(ApplyColorMapToOutlineInfo, false)
 {
     const Parameters p = codegen::bake<Parameters>(dictionary);
 
+    pointColor.setViewOption(properties::Property::ViewOptions::Color);
+    addProperty(pointColor);
+
+    addProperty(enableOutline);
+
+    outlineColor.setViewOption(properties::Property::ViewOptions::Color);
+    addProperty(outlineColor);
+
+    addProperty(outlineWeight);
+
+    outlineStyle.addOption(OutlineStyle::Round, "Round");
+    outlineStyle.addOption(OutlineStyle::Square, "Square");
+    outlineStyle.addOption(OutlineStyle::Bottom, "Bottom");
+    outlineStyle = OutlineStyle::Round;
```

Comment:
In Alex's previous comment he suggested that the default should be square, should we change that here?

Reply:
See reply in Alex's previous comment.  My intuition tells me that the default value should always be round rather than square